lib/Backend/BackendUtils.cpp

```diff
@@ -187,9 +293,17 @@ runtime::RuntimeBundle::create(const IRFunction &F,
   }
   auto constantMaxSize = constantAllocator.getMaxMemoryUsage();
 
+  // Placeholders should be allocated in a order of Input|InputOutput|Output.
+  std::vector<std::tuple<const Placeholder *, bool, bool>>
+      contiguousPlaceholders =
+          glow::getContiguousPlaceHolder<IRFunction,
```

Comment:
Is it necessary to explicitly provide the type parameters for the template function?

Reply:
> Is it necessary to explicitly provide the type parameters for the template function?

I was thinking about making it more clear because of the std::tuple used. Since we switch to a struct, I just replace it with `auto`

Reply:
I meant do we need to use `glow::getContiguousPlaceHolder<IRFunction, ...>(...)` vs just `glow::getContiguousPlaceHolder(...)` without specifying the types of template parameters? Also, why do you need to explicitly use the `glow` namespace prefix?

Your idea about using `auto` for `contiguousPlaceholders` also makes sense.

Reply:
> I meant do we need to use `glow::getContiguousPlaceHolder<IRFunction, ...>(...)` vs just `glow::getContiguousPlaceHolder(...)` without specifying the types of template parameters? Also, why do you need to explicitly use the `glow` namespace prefix?
> 
> Your idea about using `auto` for `contiguousPlaceholders` also makes sense.

I just deleted the template parameters.